### ISSUE TITLE
need caching turned off for future beta version builds

### ISSUE THREAD

**Issue by ssareli** [htmlxprs/ionic-parse-todo-app#2]
Great demo!  However, there's an issue in leveraging this example going forward.

I was testing out your app but when I tried to assemble it piece by piece my version wouldn't read from Parse without a webpage reload.  I finally figured out that they added caching in beta 14.  When I added cache: false to the todos.html controller it fixed.

**Comment by bennett-elder** [comment]
Agreed. I had to add "cache: false," to this section or the list on the main screen wouldn't refresh.
$stateProvider.state('todos',{
          cache: false,
          url:'/todos',
           controller:'TodoListController',
           templateUrl:'views/todos.html'
        })
